samples/matter/bridge/src/bridge_manager.cpp

```diff
@@ -0,0 +1,392 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "bridge_manager.h"
+
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/util/generic-callbacks.h>
+#include <lib/support/Span.h>
+#include <lib/support/ZclString.h>
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace ::chip;
+using namespace ::chip::app;
+
+BridgeManager BridgeManager::sBridgeManager;
+
+void BridgeManager::Init()
+{
+	/* The first dynamic endpoint is the last fixed endpoint + 1. */
+	mFirstDynamicEndpointId = static_cast<chip::EndpointId>(
+		static_cast<int>(emberAfEndpointFromIndex(static_cast<uint16_t>(emberAfFixedEndpointCount() - 1))) + 1);
```

Comment:
Maybe this is how it's supposed to be, but it looks a bit weird that first we subtract and then add 1 immediately :)

Reply:
Yeah I know, but this is how it's supposed to be :) First we get number of fixed endpoints (e.g. 3) and then we want to get endpoint ID object of the last fixed endpoint, by passing its index as an argument. However the indexes are assigned starting from 0, so this is where -1 comes from. After we got endpoint ID of the last fixed endpoint, we have to add +1, to set starting point of dynamic endpoints just after it.

Reply:
We cannot just get endpoint ID of the dynamic endpoint immediately (e.g by skipping -1), because at this point (init), no dynamic endpoint is created yet.